src/Hydrator/MoneyHydrator.php

```diff
@@ -20,7 +20,7 @@ class MoneyHydrator implements HydratorInterface
     public function extract($object)
     {
         return [
-            'amount' => $object->getAmount(),
+            'amount' => $object->getAmount() / 100,
```

Comment:
According to the provided list, CLP (chilean peso) is a zero-decimal currency, but it's not! There are 100 Centavos for 1 CLP. However GNF (Franc Guinéen) is valid.